sos/plugins/insights.py

```diff
@@ -13,16 +13,16 @@ class RedHatInsights(Plugin, RedHatPlugin):
     '''Collect config and log for Red Hat Insights
     '''
     plugin_name = 'insights'
-    packages = ['redhat-access-insights']
+    packages = ['redhat-access-insights', 'insights-client']
     profiles = ('system', 'sysmgmt')
-    conf_file = '/etc/redhat-access-insights/redhat-access-insights.conf'
+    conf_file = ('/etc/insights-client/insights-client.conf', '/etc/redhat-access-insights/redhat-access-insights.conf')
```

Comment:
This line looks waaaaaaaayy too long. Take a look at the Travic.CI links in the pull request and you can see the full results.

You may want to install `pycodestyle` locally, and run it from the root of your git tree as:

```
$ pycodestyle sos
```

To save time and get any errors immediately without having to wait to push.